### PR TITLE
Work card popup window

### PR DESCRIPTION
**Pull request by ShinhyoBelok**
## Changes ♻️ 
* Add a new card for the popup window in the index.html file.  🧾 ✔️ 
* Add style to the popup card. 😎 ✔️ 
* Add JavaScript functionality for the popup card. 👍 ✔️ 
* Add dynamic content.✔️ 
🎊

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -50,28 +50,29 @@ <h3 class="social">LET’S CONNECT</h3>
         </section>
         <section id="work">
             <!-- project 1 -->
-            <section class="project">
+            <section class="project" id="project-1">
+                <!-- <button class="close-btn-modal d-none-modal"><img src="img\Icon-Cancel-detailmode.svg" alt="X"/></button> -->
```
Comment (danmainah) [inline index.html]: - [ ] @team it's not good practice to have commented code in your codebase, kindly remove this commented code.

File: index.html [hunk 2]
```diff
@@ -50,28 +50,29 @@ <h3 class="social">LET’S CONNECT</h3>
         </section>
         <section id="work">
```
Comment (danmainah) [inline index.html]: - [ ] @team great work in your popup however the entire work section is supposed to be generated dynamically using javascript not hard coded as you have done, you have already created an array, just using the same data to populate this section, here are examples, eg [example1](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96) or [example2](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by fherrerao — APPROVED** [review]
# ✅  🎆  STATUS APPROVED! ✅ 🎆 
#
Hi @GutemaG, @ShinhyoBelok, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Great Job Yes GIF by Hollywood Suite


## Highlights
✔️  No linter errors.
✔️  You used the correct GitHub-Flow.
✔️  Good README
✔️  Good Commit Messages

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**